api/src/loaders/mongoose.js

```diff
@@ -4,21 +4,24 @@ const connectionString = process.env.DB_CONN_URL;
 const LOGGER = require("../common/logger");
 
 const connectToMongoose = async () => {
-    await mongoose.connect(connectionString, {
-        useNewUrlParser: true,
-        useUnifiedTopology: true,
-        useCreateIndex: true,
-        useFindAndModify: false,
-    });
-    const db = mongoose.connection;
+    //temp workaround to not connect to local db when testing
+    if (process.env.NODE_ENV !== "test") {
```

Comment:
Do you need to update programme tests to have this field?

Reply:
only in the programme-route-tests which has not been created yet. Those tests will be created in another PR

Reply:
ah ok